src/e2e/puppeteer/__tests__/render-thoughts.ts

```diff
@@ -167,7 +168,7 @@ describe('Font Size: 13', () => {
     await click('[data-testid=decrease-font]') // 13
 
     // close alert
-    await click('[data-testid=close-button]')
+    await hide('[data-testid=alert]')
```

Comment:
Could we add an ✗ that appears in the upper right corner on hover to dismiss the alert? I realize the mockup did not account for the close functionality on desktop.

I'm okay if this is + time.